lib/peep.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'posted_peeps_test')
+    else
+      connection = PG.connect(dbname: 'posted_peeps')
+    end
+
+    result = connection.exec('SELECT * FROM peeps;')
+    result.map { |peep| peep['peep'] }
+  end
+
+  def self.post(text:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'posted_peeps_test')
+    else
+      connection = PG.connect(dbname: 'posted_peeps')
+    end
+
+    connection.exec("INSERT INTO peeps (peep) VALUES ('#{text}') RETURNING peep")
```

Comment:
What happens if they make a post with a bunch of single quotation marks in them?